02-introducing-survey-data.Rmd

```diff
@@ -1,2 +1,188 @@
+
+Almost every survey conducted will have some of each of these types of error, and researchers attempt to conduct a survey that reduces this *total survey error*. One of the most common ways to reduce total survey error is through weighting. However, weighting may result in increased error on its own (*adjustment error*). Therefore when designing surveys, researchers should carefully think about ways to reduce each of these error sources. Additionally when analyzing survey data, understanding decisions that researchers took to minimize these error sources can impact how results are interpreted. The remainder of this chapter dives into considerations for survey development and places where each of these sources of error can be considered.
+
+## Study Design {#overview-design}
+
+Study design is often used to encompass multiple parts of the survey planning process including decisions on population, survey mode, and timeline. Multiple decisions made prior to the launch of the survey can assist researchers in reducing different error sources, and are fundamental in understanding how to interpret the results of the data. Knowing who and how to survey individuals depends on both the goals of the study and the feasibility of implementation.
+
+### Sampling Design {#overview-design-sampdesign}
+
+Who we want to survey is known as the *population* of interest in the study. This could be broad, such as, "all adults age 18+ living in the U.S.", or it could be a very specific population of interest based on a specific characteristic or location. For example, we may want to know about "adults age 18-24 who live in North Carolina" or "eligible voters living in Illinois". However, to survey individuals in these populations, a *sampling frame* is needed with contact information. If researchers are looking at eligible voters, the sampling frame would be the voting registry for a given state or area. For more broad populations like all adults living in the U.S., the sampling frame will most likely be imperfect. In these cases, researchers may choose to use a sample frame of mailing addresses and send the survey to households, or they may choose to use random digit dialing (RDD) and call random phone numbers. These imperfect sampling frames can result in *coverage error* where there is a mismatch between the population of interest and the list of individuals researchers can select. For example, if a researcher is looking to obtain estimates for "all adults age 18+ living in the U.S.", using a sample frame from mailing addresses will be missing specific types of individuals such as the homeless or transient populations. Additionally, many households have more than one adult living there, so researchers would need to consider how to get a specific individual to fill out the survey (called within household selection), or adjust the population to report on "U.S. households" instead of "individuals".
```

Comment:
```suggestion
Who we want to survey is known as the *population of interest*^[This is also referred to as the target population in some literature] in the study. This could be broad, such as, "all adults age 18+ living in the U.S.", or it could be a very specific population of interest based on a specific characteristic or location. For example, we may want to know about "adults age 18-24 who live in North Carolina" or "eligible voters living in Illinois". However, to survey individuals in these populations, a *sampling frame* is needed with contact information. If researchers are looking at eligible voters, the sampling frame could be the voting registry for a given state or area. For more broad populations like all adults living in the U.S., the sampling frame will most likely be imperfect. In these cases, researchers may choose to use a sample frame of mailing addresses and send the survey to households, or they may choose to use random digit dialing (RDD) and call random phone numbers. These imperfect sampling frames can result in *coverage error* where there is a mismatch between the population of interest and the list of individuals researchers can select. For example, if a researcher is looking to obtain estimates for "all adults age 18+ living in the U.S.", using a sample frame from mailing addresses will be missing specific types of individuals such as the homeless or transient populations. Additionally, many households have more than one adult living there, so researchers would need to consider how to get a specific individual to fill out the survey (called within household selection), or adjust the population to report on "U.S. households" instead of "individuals".
```